hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java

```diff
@@ -118,6 +126,11 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       validateBucketAndVolume(omMetadataManager, volumeName, bucketName);
 
+      // We do not check if this transaction is a replay here to avoid extra
+      // DB reads. Even if this transaction is replayed, in
+      // S3MultipartUploadComplete request, we would delete this entry from
```

Comment:
Can we mention this as TODO? (As still it is not addressed)

Reply:
Added a TODO in MultipartComplete request.